src/client/game/BulletPool.js

```diff
@@ -0,0 +1,29 @@
+var NUM_BULLETS = 10;
```

Comment:
Number of bullets on screen at a time. We only construct 10 bullets, and re-use them when they go off screen so we're not constructing new objects every time the user fires.